src/Views/AppListView.vala

```diff
@@ -60,13 +61,17 @@ namespace AppCenter.Views {
         private void add_row_for_package (AppCenterCore.Package package) {
             // Don't show plugins or fonts in search and category views
             if (!package.is_plugin && !package.is_font) {
-                list_store.insert_sorted (package, (GLib.CompareDataFunc<AppCenterCore.Package>) compare_packages);
+                GLib.CompareDataFunc<AppCenterCore.Package> sort_fn = (a, b) => {
```

Comment:
The reason for this lambda is I had to make compare_packages non-static so it can see the current_search_term. Attempting to cast a class's member function to GLib.CompareDataFunc<AppCenterCore.Package> makes it crash. Wrapping it in a lambda makes it safe to call the member function.